README.md

```diff
@@ -15,12 +15,13 @@ møte organises and serves meetings as a drop-in replacement, without needing an
 ### Using møte
 
 Dependencies:
- - `memcached` (`sudo dnf install memcached`)
- - Python 3.8.X
- - Pipenv
+ - [`memcached`](https://memcached.org) (`sudo dnf install memcached`)
+ - Python 3.8.x
+ - [Pipenv](https://docs.pipenv.org/) (`sudo dnf install pipenv`)
 
-Installation:
- - `pipenv install` will create a new virtual environment and install the requirements. `
+Virtual Environment setup:
+ - `pipenv install` will create a new virtual environment and install the requirements.
+ - `pipenv shell` will activate Python's virtual environement's shell.
```

Comment:
An ordered list maybe? (Because they need to be executed one after the other.)

Reply:
@t0xic0der Thanks for the review, personally I feel, It's pretty intuitive with unordered lists to follow it step by step and the list representation seems simple as points, Do you think adding an ordered list would make it more simple ?

Reply:
Either could work. There are just two points anyway. :-)